gateway-getting-started-guide.md

```diff
@@ -60,13 +63,13 @@ Select your home network from the list and enter your Wi-Fi password to connect.
 
 ### 5. Choose Subdomain
 
-Once you have connected the Pi to your wireless network, you should ensure that your laptop/tablet/smartphone is connected to the same Wi-Fi network and then navigate to **http://gateway.local** in your web browser.
+After you've connected the RPi to your wireless network, you should ensure that your laptop/tablet/smartphone is connected to the same Wi-Fi network and then navigate to **http://gateway.local** in your web browser.
```

Comment:
RPi should probably be lengthened to Raspberry Pi.

Reply:
Yep, done.